content/en/news/releases/1.13.x/announcing-1.13/change-notes/index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Istio 1.13 Change Notes
+linktitle: 1.13.0
+subtitle: Minor Release
+description: Istio 1.13.0 change notes.
+publishdate: 2021-02-08
```

Comment:
Note the date here is 2021, and should be 2022.

Reply:
Fixed.